packages/web-components/@sfx/products/src/products-base.ts

```diff
@@ -35,19 +39,23 @@ export default class ProductsBase extends LitElement {
     }
   }
 
+  protected renderStyles(): CSSResult {
+    return css`
```

Comment:
I can't find proper documentation for this. Can you link to something? Is there a particular benefit in separating these out of the render function? The child classes that override this function seem to include the same functionality.

More specifically, is there a reason why the `sfx-product` styles didn't make it into this function?

Reply:
> Can you link to something?

* https://lit-element.polymer-project.org/guide/styles
* https://lit-element.polymer-project.org/api/modules/_lib_css_tag_.html#css
* https://github.com/Polymer/lit-html/issues/823

It's meant for the static `styles` property, but more generally, it can be used if the template string is CSS instead of HTML. This is better than using `html` because it shouldn't be parsed as HTML.

> Is there a particular benefit in separating these out of the render function?

The benefit here is two-fold:

1. Classes that extend `ProductsBase` won't have to re-implement the render function just to change the styles. They only need to override the `renderStyles` function.
2. `ProductsBase` doesn't have to know about every component that extends it and list them all in the selector. It only needs to be concerned about itself.

> More specifically, is there a reason why the `sfx-product` styles didn't make it into this function?

I considered the `sfx-product` styles to be structural to the component and any component that extends it, but I'm not set on this opinion.

Reply:
Alright, thanks. I believe we were looking for alternatives on what to do with those product styles anyway, so we can leave them as-is.